packages/core/components/EditMetadata/NewAnnotationPathway.tsx

```diff
@@ -122,6 +144,52 @@ export default function NewAnnotationPathway(props: NewAnnotationProps) {
         }
     };
 
+    const getFieldNameValidationErrors = (value: string) => {
```

Comment:
This is named as a getter but has no return and instead does some side-effects.

Reply:
tentatively renamed to validateFieldName in [bc3f0b8](https://github.com/AllenInstitute/biofile-finder/pull/431/commits/bc3f0b898289c3fe9f93ca8a80e5121e7780b9b1)